lib/composer/package/loader/project_attributes_loader.rb

```diff
@@ -0,0 +1,56 @@
+module Composer

```

Comment:
Thank you. I was missing the :: root namespace and was indeed running into a namespace conflict. Will have a go at it tommorow.